packages/quick_actions/pubspec.yaml

```diff
@@ -16,17 +16,16 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0-nullsafety
+  meta: ^1.3.0
 
 dev_dependencies:
-  test: ^1.16.0-nullsafety
-  mockito: ^5.0.0-nullsafety.0
```

Comment:
That's the easiest case I've seen of removing Mockito dependencies during migration ;)

Reply:
haha it's just not used as all.